src/error.js

```diff
@@ -194,7 +194,10 @@ export function reportError(error, opt_associatedElement) {
     }
 
     // Report to console.
-    if (self.console) {
+    if (
```

Comment:
Should we still show the errors in the `localDev`?

Reply:
Yes, done.